packages/amplify-schema-validator/package.json

```diff
@@ -27,7 +27,9 @@
     "ts-node": "^8.5.0"
   },
   "dependencies": {
-    "graphql": "^16.6.0"
+    "aws-sdk": "^2.1141.0",
+    "graphql": "^15.5.0",
+    "npm-check-updates": "^16.1.0"
```

Comment:
Why the change to `aws-sdk` and `npm-check-updates`?

Reply:
reverted the changes